src/Core/ImageMap.cs

```diff
@@ -251,6 +251,29 @@ public bool TryFindItemExact(Address addr, out ImageMapItem item)
 		{
             return Items.TryGetValue(addr, out item);
 		}
+	    
+	    
+        private bool _mapChangedEventHandlerPaused;
+	    private bool _mapChangedPendingEvents;
```

Comment:
Names of fields should not be started with underscore

Reply:
Is there code style definition available somewhere?
AFAIU spaces should be used instead of tabs, right?

Reply:
Yes, we prefer spaces to tabs. However, there is no coding style guide yet. As we attract more contributors, it becomes necessary to write one. I will open an issue to track this.